### ISSUE TITLE
Add: tap to close image 

### ISSUE THREAD

**Issue by Kmad666** [K1rakishou/Kuroba-Experimental#803]
When viewing an image, trying to close it by swiping up isn't reliable, or comfortable. I'd rather tap to close like on clover.

**Comment by fhtdtdj** [comment]
+1. Please.

**Comment by K1rakishou** [comment]
Tap is already used for toggling fullscreen mode (like in 99% of other media viewers).

**Comment by Kmad666** [comment]
> +1. Please.
Yes! Please and thank you, Great work on the app. 

> Tap is already used for toggling fullscreen mode (like in 99% of other media viewers).
Maybe this will clarify what I mean:
If I'm Tapping the thumbnail of an image, and it opens the image into full screen mode, tapping the image itself doesn't close full screen mode. 
To close an image from full screen mode you have to swipe up , or hit the back button. 
So what I'm proposing is that taping the fully loaded image in full screen mode would close it.

**Comment by K1rakishou** [comment]
Full screen mode is when Android navigation UI is hidden (status/navigation bars). Single tap toggles it. This behavior is used in pretty much all media viewers.

**Comment by Kmad666** [comment]
I see, well the last 4chan app I used was blue clover, which is currently having captcha issues. 
Like you said, tapping an image in kurobaex opens it into full screen mode. 
In blue Clover full screen mode I can tap the image to close it. I can't do this for kurobaex.

**Comment by K1rakishou** [comment]
Yeah, because Clover (Blue/Red/neither of them) doesn't use fullscreen mode. You are confusing fullscreen (or immersive) mode and a media viewer.
See https://developer.android.com/develop/ui/views/layout/immersive

**Comment by fhtdtdj** [comment]
Please consider reverting to the Clover format or at least adding an option to choose it.

The most common complaint re: KurobaEX on /g/ is its myriad of small UI annoyances and unwanted modifications compared to Clover and Kuroba. With other apps getting deprecated and EX adding great new features under the hood, more will move to EX but it is a shame that UI is handled as an afterthought despite recurring objective criticism... Please listen to the community as this isn't just OP and me complaining about this.

Please don't just address this with "but other media viewers do this"; most people do not primarily use Clover forks for enhanced image viewing but just want to quickly and conveniently check out a post's attached picture. Tapping is quick and convenient, swiping is awkward and annoying.

Full-screen viewer may make sense for boards/threads dedicated to image dumps (a smaller part of the site); not for viewing discussions with casual image posting (the majority of the site and what people use mobile posting for). It is therefore only logical and expected that the quick and convenient method should be the default.

**Comment by Kmad666** [comment]
Yes, what fhtdhshd said. 
The myriad of features of kurobaex has won me over, since blue clover is expected to be kill for now I have no choice but to chose this app, and although it succeeds in many places including the excellent support for plugins and sound images. 
The most important aspect of browsing an image board is still just about opening and closing images. Swiping is clunky. Just add the enhancement to close an image by tapping it.
I'm not a programmer, which is why I can only kindly ask, but surely there is a way to add this? I mean you added a swipe to the media viewer, adding tape to close would be equivalent right?

**Comment by K1rakishou** [comment]
>compared to Clover and Kuroba

BlueClover is already fixed, Kuroba is going to be fixed soon. Everyone is going to be happy using whatever they want so there is no need to change anything.

**Comment by K1rakishou** [comment]
Actually, I think if someone sends me a PR, then I will merge it. I just don't want to do it myself.
